tests3/sqlservertests.py

```diff
@@ -462,6 +462,13 @@ def test_fast_executemany_to_datetime2(self):
         self.cursor.executemany(sql, params)
         self.assertEqual(self.cursor.execute("SELECT CAST(dt2 AS VARCHAR) FROM ##issue540").fetchval(), '2019-03-12 10:00:00.12')
 
+    def test_high_unicode(self):
+        v = "🎥"
```

Comment:
This is essentially cosmetic, but would it make it clear that this is a high unicode character if it was defined as `v = "\U0001F3A5"  # Movie Camera Emoji U+1F3A5` rather than just `v = "🎥"`?

Reply:
I agree.  Since I'd like to make a release immediately, I'll accept it as is and we can make a change afterwards.